sonar-db/src/main/java/org/sonar/db/purge/PurgeDao.java

```diff
@@ -63,13 +63,15 @@ public void purge(DbSession session, PurgeConfiguration conf, PurgeListener list
     deleteAbortedAnalyses(rootUuid, commands);
     deleteDataOfComponentsWithoutHistoricalData(session, rootUuid, conf.scopesWithoutHistoricalData(), commands);
     purgeAnalyses(commands, rootUuid);
-    disableOrphanResources(rootUuid, session, mapper, listener);
+
+    // FIXME to be re-enabled with 
+    //disableOrphanResources(rootUuid, session, mapper, listener);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)